File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        
        vector<int> result{nums[0]};
        
        for(int i = 1; i < nums.size(); i++) {
            int currentSum = result[i-1] + nums[i];
            result.push_back(currentSum);
        }
        
        return result;
    }
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        result = [nums[0]]
        
        for i in range(1, len(nums)):
            current_sum = nums[i] + result[i-1]
            result.append(current_sum)
        return result
        
        